### PR TITLE
BDR fix for service alias

### PR DESCRIPTION
**Pull request by vibhorkum**
Adding the fix for Postgres service alias

### CODE REVIEW

File: edbdeploy/data/tpaexec/hooks/post-deploy.yml [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
 
+- name: "Get the information of {{ pg_systemd_alias }} service"
+  stat:
+    path: "/etc/systemd/system/edb-as-{{ postgres_version }}.service"
+  when:
+    - "'bdr' in role"
+    - "'primary' in role"
+    - pg_systemd_alias is defined
+    - pg_systemd_service_path is defined
+  register: pg_service_stat
```
Comment (jt-edb) [inline edbdeploy/data/tpaexec/hooks/post-deploy.yml:11]: This variable does not look to be used further.

**Review by jt-edb — COMMENTED** [review]
LGTM. Only one comment.

File: edbdeploy/data/tpaexec/hooks/post-deploy.yml (hunk 1 above)
Comment (vibhorkumar123) [reply edbdeploy/data/tpaexec/hooks/post-deploy.yml]: fixed

**Review by jt-edb — APPROVED** [review]
LGTM

**Pull request merged**